src/environments/environment.ci.ts.template

```diff
@@ -33,6 +33,9 @@ const commonBaseURL = `${COMMON_API}`;
 const fhirBaseUrl = `${FHIR_API}`;
 const sessionStorageEncKey = '<%= SESSION_STORAGE_ENC_KEY %>';
 
+const siteKey = '<%= siteKey %>';
```

Comment:
Hey these are environment variables.
let's use upper case.

See how builds happen.

Refer to package.json scripts.
The ci template is processed by rebuild script in scripts folder.
It injects all the environment variables into the template and prepares environment.ci.ts file.

Please add the environment variables with empty values ci prebuild.js